### ISSUE TITLE
[Dec 2025] Accessibility Audit: Bulk upload patient user flows

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#9353]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the bulk patient upload user flow. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/upload-patients`

Use cases to test:
* successful bulk upload (inc success toast)
* bulk upload produces validation messages
* bulk upload produces error toast

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.
Export scan results as a csv and add the file to this ticket.

### Additional context
See tickets from previous audits #8515 #5781, #6331
